lib/WorkflowEpitopeprediction.groovy

```diff
@@ -37,4 +39,33 @@ class WorkflowEpitopeprediction {
         return yaml_file_text
     }
 
+    public static String methodsDescriptionText(run_workflow, mqc_methods_yaml) {
+        // Convert  to a named map so can be used as with familar NXF ${workflow} variable syntax in the MultiQC YML file
+        def meta = [:]
+        meta.workflow = run_workflow.toMap()
+        meta["manifest_map"] = run_workflow.manifest.toMap()
+
+        meta["doi_text"] = meta.manifest_map.doi ? "(doi: <a href=\'https://doi.org/${meta.manifest_map.doi}\'>${meta.manifest_map.doi}</a>)" : ""
+        meta["nodoi_text"] = meta.manifest_map.doi ? "": "<li>If available, make sure to update the text to include the Zenodo DOI of version of the pipeline used. </li>"
+
+        def methods_text = mqc_methods_yaml.text
+
+        def engine =  new SimpleTemplateEngine()
+        def description_html = engine.createTemplate(methods_text).make(meta)
+
+        return description_html
+    }//
+    // Exit pipeline if incorrect --genome key provided
+    //
+    private static void genomeExistsError(params, log) {
```

Comment:
I guess that is nf-core related and we just keep it for no conflicts with future templates since we dont have params.genome in nextflow.config?

Reply:
Yes, I think we can just leave it in there.